xroad-catalog-persistence/src/test/java/fi/dvv/xroad/catalog/persistence/repository/OrganizationRepositoryTest.java

```diff
@@ -74,7 +77,7 @@ public void testFindLatestFetched() {
         LocalDateTime latestFetched = organizationRepository.findLatestFetched();
         assertEquals(2016, latestFetched.getYear());
         assertEquals(Month.JANUARY, latestFetched.getMonth());
-        assertEquals(1, latestFetched.getDayOfMonth());
+        assertEquals(2, latestFetched.getDayOfMonth());
```

Comment:
Why did this change?

Reply:
The pipeline kept failing for some reason. So I had to change the day of the month in tests data in the `data.sql`.